src/translations/en.json

```
@@ -2671,6 +2671,14 @@
             "exclusion_failed": "The node could not be removed. Please check the logs for more information.",
             "exclusion_finished": "Node {id} has been removed from your Z-Wave network."
           },
+          "remove_failed_node": {
+            "title": "Remove a Failed Z-Wave Device",
+            "introduction": "Remove a failed device from your Z-Wave network. Use this if you are unable to exclude a device normally because it is broken.",
+            "remove_device": "Remove Device",
+            "in_progress": "The device removal is in progress.",
+            "removal_finished": "Node {id} has been removed from your Z-Wave network.",
```

```suggestion
            "removal_finished": "Node {id} has been removed from your Z-Wave network. If you still have access to the node, make sure to reset it to factory defaults.",
```
might avoid some troubleshooting if users decide to replace a working, but sleeping device or put a dead one back after they have removed it.